src/core_ct/core.py

```diff
@@ -281,3 +281,63 @@ def chunk(self, x1=0, y1=0, z1=0, x2=None, y2=None, z2=None) -> Core:
 
         new_core = Core(self.pixel_array[x1:x2, y1:y2, z1:z2], self.pixel_dimensions)
         return new_core
+
+    def shape(self) -> tuple[int, int, int]:
+        """
+        Get the pixel dimensions of the core scan.
```

Comment:
Because `pixel_dimensions` is already an attribute of the core that refers to something completely different than this, it feels confusing to say that the shape function returns the pixel dimensions of the scan. Maybe change to "get the dimensions of the pixel array of the core scan" or something similar.